src/graph/parser.ts

```diff
@@ -273,13 +294,15 @@ export class Graph {
                         break;
                     }
 
+                    case "skipCache": {
+                        graphSettings.skipCache = value ? (value.toLowerCase() == "true") : false
+                    }
+
```

Comment:
For the sake of backwards compatibility, I don't think this field should be exposed to the user at all, especially as there will likely be changes to the caching behavior in the future. So I think it would be best if this was left as an internal flag.